packages/shared/models/ai/API.md

```diff
@@ -1,30 +1,41 @@
 # Models
 
-Comprehensive **Zod schemas and TypeScript types** for Code PushUp configuration, reports, and data validation.
+Simple **TypeScript types** for Angular MCP toolkit shared interfaces and utilities.
 
 ## Minimal usage
 
 ```ts
-import { auditSchema, reportSchema } from '@push-based/models';
-
-const audit = auditSchema.parse({
-  slug: 'performance-audit',
-  title: 'Performance Audit',
-});
-
-const isValidReport = reportSchema.safeParse(reportData).success;
+import { type CliArgsObject, type ToolsConfig } from '@code-pushup/models';
+
+// CLI argument types
+const args: CliArgsObject = {
+  directory: './src',
+  componentName: 'DsButton',
+  _: ['command'],
+};
+
+// MCP tool configuration
+const toolConfig: ToolsConfig = {
+  schema: {
+    name: 'my-tool',
+    description: 'A custom tool',
+    inputSchema: {
+      type: 'object',
+      properties: {},
+    },
+  },
+  handler: async (request) => {
+    return { content: [{ type: 'text', text: 'Result' }] };
+  },
+};
 ```
```

Comment:
Do we need this kind of docs here?? I suggest tests or as second place JSDocs at the code place

Reply:
Yes, we do, it is usage examples for ai